src/core/distributions/phylogenetics/substitution/AbstractPhyloCTMCSiteHomogeneous.h

```diff
@@ -2077,11 +2080,11 @@ void RevBayesCore::AbstractPhyloCTMCSiteHomogeneous<charType>::tipDrawJointCondi
             double sum = 0.0;
 
             // if the matrix is compressed use the pattern for this site
-            size_t pattern = i;
-            if ( compressed == true )
-            {
-                pattern = site_pattern[i];
-            }
+            // size_t pattern = i;
```

Comment:
is this change intentional ? I don't see why it's needed
either way it would probably be better to just remove those lines than comment out

Reply:
This suppresses a compiler warning about an initialized but unused variable.  I'll remove the line completely.